src/tests/intg/test_enumeration.py

```diff
@@ -465,7 +465,7 @@ def _test_add_remove_membership_rfc2307(ldap_conn, user_and_group_rfc2307):
 
 
 def _test_add_remove_membership_rfc2307_bis(ldap_conn,
-                                           user_and_groups_rfc2307_bis):
+                                            user_and_groups_rfc2307_bis):
```

Comment:
NACK.
You would need to change this back after reverting a24e735d33b788fec32acba70f30709eff44ebdf.
Because it is blocker for 1.15.4